docs/source/conf.py

```diff
@@ -0,0 +1,64 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'osier'
+copyright = '2022, Samuel Dotson'
+author = 'Samuel Dotson'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "myst_parser",
+    "numpydoc",
+    "sphinxcontrib.napoleon",
```

Comment:
Thanks for the `napoleon` update! Regarding the two warnings you receive, they're both ignorable for now. 
* The _static folder doesn't exist because I don't have any static files and git won't let you push an empty directory. 
* The `myst` reference warning is a bit strange and may require a separate issue. The explanation is that I prefer to have a single contrib file rather than a contrib file in the docs that references another contributing file. I'm not sure what the best way to handle this is, the only difference is that the README has a hyperlink to the contrib file when viewed on github, while the docs has a hyperlink immediately below the reference.